src/Header/Account/Account.module.scss

```diff
@@ -122,4 +122,9 @@
       text-decoration: underline;
     }
   }
+
+  span {
```

Comment:
Can we merge the common bits between the span and a into a single declaration so they're more obvious they're styling the same element and so they're definitely the same values - avoids them getting out of sync if someone changes the margin on one but not the other.